docker/push.sh

```diff
@@ -15,7 +15,11 @@ git describe --tags --dirty --always > .git-version
 
 TAG=${1:-beta}
 echo "building for tag $TAG"
-docker build -t metabrainz/listenbrainz:"$TAG" \
+docker build \
+        --cache-from metabrainz/listenbrainz:latest \
```

Comment:
should we only tag `:latest` if TAG is a v-[date], compared to beta/test/etc?

Reply:
I think that makes sense to do.